packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -39,34 +40,60 @@ const OUTPUT_PATH = path.join(LOG_PATH, 'output');
 const SCREENSHOTS_PATH = path.join(LOG_PATH, 'screenshots');
 const COVERAGE_PATH = path.join(LOG_PATH, 'coverage');
 
-// mongodb-runner defaults to stable if the env var isn't there
-export const MONGODB_VERSION = (process.env.MONGODB_VERSION || '5.0.6')
-  // semver interprets these suffixes like a prerelease (ie. alpha or rc) and it
-  // is irrelevant for our version comparisons anyway
-  .replace('-community', '')
-  .replace('>', '')
-  // HACK: comparisons don't allow X-Ranges and 5.x or 5.x.x installs 5.2.1 so
-  // we can't just map it to 5.0.0
-  .replace(/x/g, '999');
+let MONGODB_VERSION = '';
+let MONGODB_USE_ENTERPRISE = process.env.MONGODB_USE_ENTERPRISE ?? 'no';
 
-export const MONGODB_USE_ENTERPRISE =
-  process.env.MONGODB_USE_ENTERPRISE ?? 'no';
+export const MONGODB_TEST_SERVER_PORT = Number(
+  process.env.MONGODB_TEST_SERVER_PORT ?? 27091
+);
 
-// This will just match any latest available release. We will need to update it
-// when next major version will start rolling out (this is not just something
-// like 999.999.999 because we are checking for `< 7.0.0-alpha` in one test)
-const LATEST_ALPHA = '6.999.999';
+export function updateMongoDBServerInfo() {
+  try {
+    const { stdout, stderr } = crossSpawn.sync(
```

Comment:
[Here's an evergreen patch](https://spruce.mongodb.com/version/63e657293e8e86348cba68dd/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) where everything but windows is green. After than I switched this part of code to `crossSpawn` so windows should pass too now